.github/workflows/package.yml

```diff
@@ -130,17 +145,17 @@ jobs:
   codesign:
     name: Codesign
     runs-on: ${{ matrix.runner }}
-    environment: publish-prod
     needs: preflight
+    environment: ${{ needs.preflight.outputs.package-env }}
     strategy:
       matrix:
         project: [ jetsocat, devolutions-gateway, devolutions-agent ]
-        os: [ windows, macos, linux ]
+        os: [ windows, linux ]
         include:
           - os: windows
             runner: windows-2022
-          - os: macos
-            runner: macos-latest
+          #- os: macos
+          #  runner: macos-latest
```

Comment:
nitpick: Feel free to remove unused code.

Reply:
This was because the MacOS runner in test mode didn't have proper keys for signing. Will update it now now that the agent packages correctly.